pkg/cmdutils/cmdutils.go

```diff
@@ -170,6 +170,10 @@ func NewBookieClient() bookkeeper.Client {
 	return PulsarCtlConfig.BookieClient()
 }
 
+func NewBookieClientFromConfig(config *ClusterConfig) bookkeeper.Client {
```

Comment:
I think we don't need this. If you want to use it in other places, you can initialize the bookkeeper directly.
```
	config := bookkeeper.DefaultConfig()
		config.WebServiceURL = c.BKWebServiceURL
	}

	bk, err := bookkeeper.New(config)
	if err != nil {
		log.Fatalf("create bookie client error: %s", err.Error())
	}
```

Reply:
OK, I didn't notice these functions.